cmd/timescale-reader/main.go

```diff
@@ -107,23 +107,23 @@ func main() {
 		return
 	}
 
-	tc, tcHandler, err := auth.SetupAuthz(ctx, authConfig)
+	thingsClient, thingsHandler, err := auth.SetupThingsClient(ctx, authConfig)
 	if err != nil {
```

Comment:
```suggestion
	thingsClient, thingsHandler, err := auth.SetupThingsClient(ctx, thinggRPCClientConfig)
```
and 
@felixgateru Should rename  the `github.com/absmach/magistrala/pkg/auth` to some thing like this  `github.com/absmach/magistrala/pkg/grpcclient`